oval_graph/command_line.py

```diff
@@ -8,13 +8,6 @@
 CEND = '\033[0m'
 
 
-def print_where_is_saved_result(results_src):
-    if results_src:
-        print("Results are saved:")
```

Comment:
If a program outputs data to be processed, it usually does that to standard output. If it displays its internal log, it usually uses standard error. So I would suggest using `print(..., file=sys.stderr)` instead.